### PR TITLE
Reserved env variables and Encryption Process

### PR DESCRIPTION
**Pull request by magreenbaum**
Included secrets encryption and information. 
Closes: https://github.com/meltano/cloud-docs/issues/10

### CODE REVIEW

File: docs/encrypting_secrets.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+> Each time encryption is performed, all contents in the .env file will update.
+
+### Example
+
+For this example, the following statements are true:
+
+- I am in my meltano project root directory
+- Public key file is named `meltano.pub`
+- Environment variable secrets are set in a file called `.env`
+- I want my `secrets.yml` encrypted file in the root of my project directory
```
Comment (WillDaSilva) [inline docs/encrypting_secrets.md:42]: This is a nice way to establish context for an example :+1:

**Review by WillDaSilva — APPROVED** [review]
Looks good to me, though there's some extra whitespace e.g. at the ends of some lines. We may want to set up a linter or auto-formatter if we intend to maintain these docs long-term. I'm not sure what the long-term plan for Meltano Cloud documentation is.

**Comment by magreenbaum** [comment]
> Looks good to me, though there's some extra whitespace e.g. at the ends of some lines. We may want to set up a linter or auto-formatter if we intend to maintain these docs long-term. I'm not sure what the long-term plan for Meltano Cloud documentation is.

@WillDaSilva Thanks and yeah that's a good idea. I'll follow up in another PR to add some linting and fix all the whitespacing at that point.

**Comment by WillDaSilva** [comment]
> I'll follow up in another PR to add some linting and fix all the whitespacing at that point.

@magreenbaum I'd recommend using `pre-commit` for this. Example: https://github.com/meltano/meltano/blob/main/.pre-commit-config.yaml

This repository is public, so we can use https://pre-commit.ci for free to automatically fix PRs, and check that the rules are being adhered to.

**Pull request merged**